graphics/scantailor-universal/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+PortGroup                       github 1.0
+PortGroup                       cmake 1.0
+PortGroup                       qt5 1.0
+
+github.setup                    trufanov-nok scantailor 0.2.7
+github.tarball_from             archive
+name                            scantailor-universal
+conflicts                       scantailor
+platforms                       darwin
+maintainers                     {raphael @raphael-st} openmaintainer
+license                         GPL-3+
+categories                      graphics aqua
+description                     A fork of the original scantailor: an interactive post-processing tool for scanned pages. 
+homepage                        http://scantailor.org/
+long_description                Scan Tailor is an interactive post-processing tool for scanned \
+                                pages. It performs operations such as page splitting, deskewing, \
+                                adding/removing borders, and others. You give it raw scans, and \
+                                you get pages ready to be printed or assembled into a PDF or \
+                                DJVU file. Scanning, optical character recognition, and \
+                                assembling multi-page documents are out of scope of this project. \
+                                This fork merges the features of the ScanTailor Featured and \
+                                ScanTailor Enhanced versions, brings new ones and fixes.
+
+
+depends_lib-append              port:jpeg \
+                                port:zlib \
+                                port:libpng \
+                                port:tiff \
+                                port:boost \
+                                port:xrender
```

Comment:
Sort these dependencies alphabetically.

Reply:
I did not ask anything tu @raphael-st, I just kept as it was :). So please, @raphael-st, let me know if you wish to stay as maintainer or not! :)

Reply:
I don't use Scan Tailor anymore, so I don't have any problems if you would like to maintain the scantailor port.